src/track/signature.h

```diff
@@ -0,0 +1,49 @@
+// signature.h
+// Created on: 06/11/2019 by Javier Vilarroig
+
+#pragma once
+
+namespace mixxx {
+
+// Musical Time signature
+// Right now is to be used for bar detection only, so only the beats per bar is useful
+class TimeSignature final {
```

Comment:
rename the file to track/timesignature.h

Reply:
Done.